content/blog/2022/01/2022-01-06-gsoc-2022.adoc

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: "Google Summer of Code 2022 call for Project Ideas and Mentors"
+tags:
+- gsoc
+- gsoc2022
+- community
+- events
+- developer
+authors:
+- alyssat
+- jmMeessen
+description: >
+  This blogpost calls for projects and mentors for the 2022 edition of Google Summer of Code.
+links:
+  discourse: true
+---
+
+link:https://summerofcode.withgoogle.com[Google Summer of Code] (GSoC) is a global, online mentoring program focused on introducing new contributors to open source software development. 
+GSoC contributors work on a 12+ week programming project with the guidance of mentors from their open source organization.  
+During GSoC, participating contributors are paired with mentors from open source organizations, gaining exposure to real-world software development techniques. 
+GSoC contributors will learn from experienced open source developers while writing code for real-world projects! 
+A small stipend is provided as an incentive.
+
+See GSoC contributor eligibility link:https://summerofcode.withgoogle.com/get-started[here].
+
+We are looking for project ideas and mentors to participate in GSoC 2022. 
+GSoC project ideas are coding projects that potential GSoC contributors can accomplish in about 12+ weeks. 
+The coding projects can be new features, plugins, test frameworks, infrastructure, etc. 
+Anyone can submit a project idea, but of course we like it even better if you mentor your project idea.
+
+Please send us your project ideas before the beginning of February so they can get a proper review by the GSoC committee and by the community. 
+
+=== How to submit a project idea
+Create a pull-request with your idea in a .adoc file in the link:https://github.com/jenkins-infra/jenkins.io/tree/master/content/projects/gsoc/2022/project-ideas[project ideas]. 
+It is not necessary to submit a Google Doc, but it will still work if you want to do that. 
+See the link:https://www.jenkins.io/projects/gsoc/proposing-project-ideas/[instructions] on submitting ideas which include an .adoc template and some examples.
+
+=== Current list of ideas
+Below is a current list of proposed project ideas for GSoC contributors to review. 
+Note that this list is still evolving and is subject to change.  
+A final list along with project detail and description is coming soon.
+
+* Jenkinsfile Runner Action for GitHub Actions (Mentors: Oleg Nenashev, Kris Stern)
+// Link needs to be corrected as soon as the GSOC 2022 PR is merged
+* link:https://www.jenkins.io/projects/gsoc/2021/project-ideas/automatic-spec-generator-for-jenkins-rest-api/[Automatic Specification Generator for Jenkins REST API] (Mentors: Kristin Whetstone)*
```

Comment:
Prefer links that are site local rather than including the www.jenkins.io in the URL

```suggestion
* link:/projects/gsoc/2021/project-ideas/automatic-spec-generator-for-jenkins-rest-api/[Automatic Specification Generator for Jenkins REST API] (Mentors: Kristin Whetstone)*
```